src/notCommand.ts

```diff
@@ -2,7 +2,7 @@ import * as vsc from 'vscode'
 import * as ts from 'typescript'
 import { invertExpression } from './utils/invert-expression'
 
-export const NOT_COMMAND = 'complete.notTemplate'
+export const NOT_COMMAND = '_postfix.applyNotTemplate'

```

Comment:
To be honest I have an idea to drop this command at all and replace this with multiple versions of the template, so instead of picking `not` template and then selecting from quick pick one would see `not#1`, `not#2` (or sth similar)

This would not only help with those tests here but also make it more friendly to the user in my opinion + less clicks to apply the template.

Reply:
Interesting idea. While it can make sense in case when I need to apply `not` on identifier (identifier `not` will always go first and it takes less clicks), but when I need to always revert whole binary expression it might be painful for users that have:

1. Other templates starting with `not` eg `notZ` will be last
2. For js only: word-suggestions

<img width="525" alt="Screenshot 2023-02-02 at 01 58 45" src="https://user-images.githubusercontent.com/46503702/216185165-eb92f36d-d668-4113-95e2-12b12fa3d152.png">

It will be hard to apply last variant, so I don't see any advantages of it (for example web storm also provides selector as second step).

But we definitely need to add `at` selector (0 or -1) support for `not` template. So I can have postfixes that always revert either identifier or whole binary epxression.

BTW I renamed it here so its not shown in keybindings.json command autocomplte and other locations as commands starting with `_` are internal.